recipes/scikit-garden/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "scikit-garden" %}
+{% set version = "0.1.3" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "d6fe65205f39633f17c181f552a0fc56e2f05bb2096be63f38c160fe20718d9d" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy 1.7.*   # [py27]
```

Comment:
Is this really true? Does scikit-garden only support ```numpy 1.7``` on Python 2?

Reply:
New docs, https://conda-forge.org/docs/meta.html#building-against-numpy

Reply:
I don't think that there is any specific requirement on the version of `numpy`, it definitely works with newer versions as well.

Reply:
@pmlandwehr in that case I recommend you to go for the configuration in https://github.com/conda-forge/staged-recipes/pull/3835#discussion_r135840331

That will ensure an old enough `numpy` that is forwards compatible up to the latest version available.

Reply:
Thanks all for working this out!